content/manager/implement-multi-tenancy.md

```diff
@@ -0,0 +1,105 @@
+---
+layout: bt_wiki
+title: Implementing Multi-Tenancy
+category: Manager Intro
+draft: false
+weight: 600
+
+terminology_link: reference-terminology.html
+workflows_link: workflows-built-in.html
+---
+
+Multi-tenancy is a function that enables you to create multiple independent logical groups as isolated environments, which can be managed by a single Cloudify Manager. A tenant is a logical entity that contains all its resources, for example, blueprint, deployment, workflows and so on. Using multi-tenancy is useful when you want to limit access to a specific set of data (the tenant) to a defined set of users.
+
+Multi-tenancy is implemented when you install Cloudify Manager.
+
+* If you are a **Cloudfiy Community Edition user**, a single built-in tenant is created during installation. The tenant is not visible to you and you cannot perform any actions on it such as adding a user or changing a password. There is a single user for the tenant, the `administrator`. When you log into Cloudify Manager, the built-in credentials are used.
+
+* If you are a **Cloudify Premium customer**, you can create new tenants and perform a variety of actions on them, such as adding users and user groups, setting passwords, assigning roles, and more. If you integrate LDAP or your AD into Cloudify Manager, you use the users and user groups defined there for access to the manager. You must have Cloudify Manager administrator permissions to perform user-related actions.
+
+##Multi-Tenancy for Premium Users##
+There are two options for using the multi-tenancy function in Cloudify Manager, as a standalone function that is not connected to an existing user managment system (LDAP/AD) or as a function that is integrated with the LDAP/AD user definitions. You must select one option only. If you already have LDAP/AD running in your organization, you probably want to use that, to streamline the process. If you do not use the LDAP/AD option, you must define each user individually.
+
+In the following diagram note that there are three tenants in Cloudify Manager. Tenants can be assigned groups of users, individual users, or a combination of both groups and individuals.
+ 
+![Multi-tenancy Options]({{< img "manager/multi-tenancy-options.png" >}})
+
+###Basic Workflow for Adding Tenants, User Groups and Individual Users###
+
+This section describes a basic workflow for adding a tenant, then adding one or more users to it. It is relevant only to Enterprise users.
+1. You must start with a tenant. When you install Cloudify, a default tenant is also installed. Either use the default tenant or create one or more new tenants.  
```

Comment:
"Must" isn't entirely true cause you can define the users first. Maybe "It's recommended to" is a better phrasing.